en/orm/behaviors/counter-cache.rst

```diff
@@ -87,3 +87,15 @@ a callback method. This callable must return the count value to be stored::
             }
         ]
     ]);
+
+
+.. note::
+
+    CounterCache behavior works for BelongsTo associations only. That is, for the
+    following setup: "Comments belongsTo Articles", you need to add CounterCache
```

Comment:
You could simplify the second sentence like:

"For example for "Comments belongsTo Articles" you need to add the CounterCache behavior to the ``CommentsTable`` in order to generate ``comment_count`` for the Articles table."